packages/x-tailwindcss/package.json

```diff
@@ -35,15 +35,39 @@
     "@empathyco/x-utils": "^1.0.0-alpha.9",
     "tslib": "~2.3.0"
   },
+  "peerDependencies": {
+    "vue-class-component": "^7.1.0",
+    "vue-property-decorator": "^8.3.0",
+    "vue": "^2.6.0"
+  },
+  "peerDependenciesMeta": {
+    "vue-class-component": {
+      "optional": true
```

Comment:
This makes the peeer dependencies optional. If you don't use the showcase components you don't need them

Reply:
Nice! I wasn't aware of this